onnx/defs/nn/defs.cc

```diff
@@ -36,34 +45,31 @@ void convPoolTypeAndShapeInference(InferenceContext& ctx, bool use_dilation, boo
     return;
   }
 
-  size_t n_input_dims = (size_t) (ctx.getInputType(0)->tensor_type().shape().dim_size() - 2);
+  auto input_shape = ctx.getInputType(0)->tensor_type().shape();
```

Comment:
Isn't that getInputType can return `nullptr`? @anderspapitto @gramalingam

Reply:
The preceding hasNInputShapes check verifies that the inputType is nonnull.